src/components/ArticleController.jsx

```diff
@@ -19,17 +19,15 @@ export default class ArticleController extends FalcorController {
       let articleSlug = this.props.params.articleSlug;
       // Access data fetched via Falcor
       const articleData = this.state.data.articlesBySlug[articleSlug];
-      //console.log("Data: " + JSON.stringify(articleData));
       return (
         <div>
-          {/*
-            <div>Controller for article: {articleData.title}</div>
-            <div>Ready?: {this.state.ready ? 'true' : 'false'}</div>
-          */}
           <Article
             title={articleData.title}
+            teaser={articleData.teaser}
+            pubDate={articleData.published_at}
             html={articleData.html}
             authors={articleData.authors}
+            url={"thegazelle.org/issue/" + articleData.issueId + '/' + articleData.category + '/' + articleData.slug}
```

Comment:
I'm just wondering if we omit the www will Facebook or Twitter consider it
a different link from the one with www? Not sure, cause Facebook collates
similar posts on the same page.

On Sun, Aug 7, 2016, 4:24 AM Zane Mountcastle <notifications@github.com>
wrote:

> In src/components/ArticleController.jsx
> <https://github.com/thegazelle-ad/gazelle-front-end/pull/41#discussion_r73794741>
> :
>
> >              html={articleData.html}
> >              authors={articleData.authors}
> > +            url={"thegazelle.org/issue/" + articleData.issueId + '/' + articleData.category + '/' + articleData.slug}
>
> It's for passing directly to Article's child SharingButtons. It's used
> when dynamically constructing the sharing buttons' links. With Twitter for
> example, I use this prop to display the URL in the body of the tweet and
> make it as succinctly as possible (i.e.
> thegazelle.org/issue/55/in-focus/sleep instead of
> https://www.thegazelle.org/issue/55/in-focus/sleep). Taking it directly
> from the DOM, I'd get the https://www in the tweet text box taking up
> more characters and making it look less clean IMO.
>
> Because of this, I'd either have to construct the link like this or trim
> the result of the DOM's URL (which would result in more time complexity).
> Becuase I use this once per share button, I figured it was best to just
> pass it down.
>
> —
> You are receiving this because you were assigned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/thegazelle-ad/gazelle-front-end/pull/41/files/a85a19c83b836058745a11879e25ef14a8a7a67e#r73794741>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADAFzZG4N0F7bNNzS7MCp8Wut3ZAfWhbks5qdUHsgaJpZM4JdXZj>
> .
>